mmdet/apis/ote/apis/detection/configuration.yaml

```diff
@@ -157,3 +157,44 @@ postprocessing:
 type: MODEL_CONFIG
 visible_in_ui: true
 workspace_id: ''
+pot_parameters:
+  description: POT Parameters
+  header: POT Parameters
+  preset:
+    affects_outcome_of: NONE
+    default_value: Performance
+    description: Quantization preset that defines quantization scheme.
+    editable: True
+    enum_name: POTQuantizationPreset
+    header: Preset
+    options:
+      MIXED: Mixed
```

Comment:
I don't think we really need to expose the preset option. It is defined on a per-model basis. I mean once the topology is known it should be defined once. I think currently this is true that we know the model architecture in advance.

Reply:
Do you mean add preset per topology?

Reply:
I mean preset is something fixed per model and user does not need to change it. So no reason to expose it as a parameter.

Reply:
Everything that need to be passed to the Task (at its initialization) should be specified as a parameter. Though there is an option to make it invisible/not-editable for the user.

Reply:
preset is hidden for user